docs/lang/articles/math/math_module.md

```diff
@@ -49,34 +47,33 @@ def test():
 
 :::note
 
-Difference between Taichi's math module and Python's built-in math module:
+Taichi's math module has a large overlap with Python's built-in math module. But you should follow a few extra rules when using Taichi's math module:
 
-- Functions in Taichi's math module must be called from within the Taichi scope.
-- Functions in Taichi's math module also accept vectors/matrices as arguments.
-- The precision of a function in Taichi's math module depends on the value of `default_fp` and backend specified in the `ti.init()` method call.
+- You must call the functions provided by Taichi's math module from within the Taichi scope.
+- Functions in Taichi's math module also accept vectors or matrices as arguments.
+- The precision of a function in Taichi's math module depends on the value of `default_fp` and the backend specified in the `ti.init()` method call.
 
 :::
 
 ## Small vector and matrix types
 
-Taichi's math module supplies a few small vector and matrix types:
+Taichi's math module provides a few small vector and matrix types:
 
 
-+ `vec2/vec3/vec4` for 2D/3D/4D floating-point vector types.
-+ `ivec2/ivec3/ivec4` for 2D/3D/4D integer vector types.
-+ `uvec2/uvec3/uvec4` for 2D/3D/4D unsigned integer vector types.
-+ `mat2/mat3/mat4` for 2D/3D/4D floating-point square matrix types.
+- `vec2/vec3/vec4` for 2D/3D/4D floating-point vector types.
+- `ivec2/ivec3/ivec4` for 2D/3D/4D integer vector types.
+- `uvec2/uvec3/uvec4` for 2D/3D/4D unsigned integer vector types.
+- `mat2/mat3/mat4` for 2D/3D/4D floating-point square matrix types.
 
-
-These vector/matrix types are created with the two template functions `ti.types.vector()` and `ti.types.matrix()`. For example, `vec2` is defined in the following way:
+To create one of the vector/matrix types above, use template function `ti.types.vector()` or `ti.types.matrix()`. For example, `vec2` is defined as follows:
```

Comment:
```suggestion
To create one of the vector/matrix types above, use template function `ti.types.vector()` or `ti.types.matrix()`. For example, `vec2` is defined as follows:
```
Do you know why I omit "the" here? Because it's not definite.